package/kernel/linux/modules/usb.mk

```diff
@@ -616,7 +616,7 @@ define KernelPackage/usb-audio
 	CONFIG_SND_USB_AUDIO
   $(call AddDepends/usb)
   $(call AddDepends/sound)
-  DEPENDS+=+kmod-media-controller
+  DEPENDS+=+kmod-media-controller +PACKAGE_kmod-sound-midi2-usb:kmod-sound-midi2
```

Comment:
It's all good just this concern me. On another project we are pushing for reproducible build. These kind of cross dependency are problematic.

Any reason why this is needed?